src/common/tracing.ts

```diff
@@ -1,12 +1,20 @@
 import { Tracing } from '@map-colonies/telemetry';
-import { ExpressInstrumentation } from '@opentelemetry/instrumentation-express';
-import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
 import { IGNORED_INCOMING_TRACE_ROUTES, IGNORED_OUTGOING_TRACE_ROUTES } from './constants';
 
-export const tracing = new Tracing([
-  new HttpInstrumentation({
-    ignoreIncomingPaths: IGNORED_INCOMING_TRACE_ROUTES,
-    ignoreOutgoingUrls: IGNORED_OUTGOING_TRACE_ROUTES,
-  }),
-  new ExpressInstrumentation(),
-]);
+const tracing = new Tracing(undefined, {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  '@opentelemetry/instrumentation-http': {
+    ignoreIncomingRequestHook: (request): boolean =>
+      IGNORED_INCOMING_TRACE_ROUTES.some((route) => request.url !== undefined && route.test(request.url)),
+    ignoreOutgoingRequestHook: (request): boolean =>
+      IGNORED_OUTGOING_TRACE_ROUTES.some((route) => typeof request.path === 'string' && route.test(request.path)),
+  },
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  '@opentelemetry/instrumentation-fs': {
+    requireParentSpan: true,
+  },
+});
+
+tracing.start();
```

Comment:
Why did you move this? Because of the `node --require`?

Reply:
yes